src/Features/ERDDAP/Platform/Observations/CurrentConditions/index.tsx

```diff
@@ -20,16 +25,88 @@ interface Props {
  *
  * @param platform Platform to display data for
  */
-export const ErddapCurrentPlatformConditions: React.FunctionComponent<Props> = ({ platform }) => (
-  <Row>
-    <WindCard platform={platform} />
-
-    <DataCard platform={platform} data_types={conditions.airTemp} />
-    <DataCard platform={platform} data_types={conditions.airPressure} />
-    <DataCard platform={platform} data_types={conditions.waveHeight} />
-    <DataCard platform={platform} data_types={conditions.wavePeriod} />
-    <DataCard platform={platform} data_types={conditions.waveDirection} />
-    <DataCard platform={platform} data_types={conditions.waterTemp} />
-    <DataCard platform={platform} data_types={conditions.visibility} />
-  </Row>
-)
+export const ErddapCurrentPlatformConditions: React.FunctionComponent<Props> = ({ platform }) => {
+  const unitSystem = useUnitSystem()
+
+  const dayAgo = aDayAgoRounded()
+
+  const airTemp = filterTimeSeries(platform.properties.readings, conditions.airTemp, dayAgo)
+  const airPressure = filterTimeSeries(platform.properties.readings, conditions.airPressure, dayAgo)
+  const waveHeight = filterTimeSeries(platform.properties.readings, conditions.waveHeight, dayAgo)
```

Comment:
Codacy found an issue: ['filterTimeSeries' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)